GameProject/Assets/OpenWorld Assets/Scripts/trdprsnCamera.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class trdprsnCamera : MonoBehaviour
+{
+    [SerializeField] Transform lookAt;
```

Comment:
Doesn't match the private field naming convention of the project. Also missing an explicit access modifier. Please rectify both.